dev/com.ibm.ws.security.wim.adapter.ldap_fat/publish/servers/com.ibm.ws.security.wim.adapter.ldap.fat.ad.ssl/server.xml

```diff
@@ -3,87 +3,41 @@
     All rights reserved. This program and the accompanying materials
     are made available under the terms of the Eclipse Public License v1.0
     which accompanies this distribution, and is available at
-    http://www.eclipse.org/legal/epl-v10.html
+    http:\\www.eclipse.org\legal\epl-v10.html
    
     Contributors:
         IBM Corporation - initial API and implementation
  -->
-<server description="new server">
+<server description="com.ibm.ws.security.registry.ldap.fat.unboundid.ad.ssl">
```

Comment:
I am not really sure why this file has many, if any changes at all. I would expect the test to be identical to what it had been.